src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -431,6 +431,42 @@ blockstore_checkpt( fd_replay_tile_ctx_t * ctx ) {
   }
 }
 
+static void
+funk_publish( fd_replay_tile_ctx_t * ctx, ulong root ) {
+  fd_blockstore_start_read( ctx->blockstore );
+  fd_hash_t const * root_block_hash = fd_blockstore_block_hash_query( ctx->blockstore, root );
+  fd_funk_txn_xid_t xid;
```

Comment:
We put the slot number in the first ulong by convention

Reply:
fd_funk_txn_xid_t xid;
      fd_memcpy(xid.uc, ctx->blockhash.uc, sizeof(fd_funk_txn_xid_t));
      xid.ul[0] = fork->slot_ctx.slot_bank.slot;